proto/messages.proto

```diff
@@ -53,11 +53,13 @@ message Text {
   repeated Mention mentions = 4;
   optional Quote quote = 5; // if this Text is part of a MessageEdit, this field is ignored
   optional bool expects_read_confirmation = 6 [default = false]; // whether the sender is expecting to receive a read confirmation
+  optional LegalHoldStatus legal_hold_status = 7; // whether this message was sent to legal hold
```

Comment:
Default will be `LegalHoldStatus.DISABLED`, since in proto2 enums "the default value is the first value listed in the enum's type", see https://developers.google.com/protocol-buffers/docs/proto#optional